docker-compose.yml

```diff
@@ -8,11 +8,3 @@ services:
       HOMEBREW_GITHUB_API_TOKEN: $GITHUB_TOKEN
     volumes:
       - ./:/home/dev.user/homebrew-core
-  test_without_rvm:
```

Comment:
I'd rename the other one from `test_with_rvm` to `cli`. It's easier to type out.

Since there's only one Dockerfile, you can rename `with_rvm.dockerfile` to just `Dockerfile`.

Reply:
Updated.